src/Tizen.BatteryMonitor/Tizen.BatteryMonitor/BatteryMonitor.cs

```diff
@@ -0,0 +1,206 @@
+﻿/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+
+namespace Tizen.BatteryMonitor
+{
+    static internal class Globals
+    {
+        internal const string LogTag = "Tizen.BatteryMonitor";
+    }
+
+    /// <summary>
+    /// Provides information related to the power consumption by applications or by hardware resources on a battery-powered device for a given duration of time.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
```

Comment:
The master branch is related to API8 changes. If your changes are going to be merged on that branch, the since _tizen tag should be `<since_tizen>8</since_tizen>`. However, in your ACR on Jira you have mentioned Platform Version 5.5, which is API7 and should be `<since_tizen>7</since_tizen>`. Please clarify the issue with the API and platform version.

Reply:
it should be <since_tizen>7</since_tizen>. I will update that. Thanks.